src/ballot-forms.php

```diff
@@ -211,141 +265,237 @@
+                                <div class="row"><!-- Start new row -->
+                            <?php endif; ?>
+                        <?php endif; ?>
+                    <?php endwhile; ?>
+                </div><!-- Close row -->
+                <div class="row justify-content-center">
+                    <div class="col-lg-12 col-md-12 col-sm-12">
+                        <hr>
+                    </div>
                 </div>
-              </div>
-            </label>
-          </div>
-          <div class="col-lg-6 col-md-12 col-sm-12 p-xl-4">
-            <label>
-              <div class="candidate-info ps-4">
-                <img src="images/candidate-profile/placeholder.png" alt="Candidate Image" width="100px" height="100px">
-                <div>
-                  <input type="radio" name="secretary-candidate">
-                  Candidate Name<br>
-                  <div class="main-color subtitle-2"><b>Section</b> </div>
+                <div class="row justify-content-center">
+                    <div class="col-lg-12 col-md-12 col-sm-12 text-center pt-2 pb-4">
+                        <label class="text-muted">
+                            <input type="radio" name="position[<?php echo $row['position_id'] ?>]" value="">
+                            <b>ABSTAIN</b><br>
+                        </label>
+                    </div>
                 </div>
-              </div>
-              <div class="pb-sm-4"></div>
-            </label>
-          </div>
-        </div>
-      </div>
+            </div><!-- Close reminder -->
+        <?php endwhile; ?>
+    <?php endif; ?>
+    <div class="text-center pb-4 mt-3">
+        <button type="button" class="button-reset" onclick="resetForm()"><u>Reset Form</u></button>
+        <button type="submit" class="button-submit main-bg-color" id="submitVoteBtn" onclick="validateForm()">
+            Submit Vote
+        </button>
     </div>
   </div>
+</form>
 
-  <div class="text-center mt-3">
-  <button type="button" class="button-reset" onclick="resetForm()"><u>Reset Form</u></button>
-  <button type="button" class="button-submit bg-color">
-    <!-- Temporary only -->
-    <a href="../src/feedback-suggestions.php"> Submit Vote </a>
-  </button>
-</div>
-</div>
+
+</main>
 </body>
 
-  
-<footer class="navbar navbar-expand-lg navbar-light bg-light mt-5">
-    <div class="container-fluid">
-      <span class="navbar-text mx-auto fw-bold spacing main-color">BSIT 3-1 | ALL RIGHTS RESERVED 2024</span>
-    </div>
-  </footer>
+<?php include_once __DIR__ . '/includes/components/footer.php'; ?>
+
 
   <script src="../src/scripts/feather.js"></script>
   <script src="../vendor/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
-  <script src="../src/scripts/script.js"></script>
   <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.4/dist/umd/popper.min.js"></script>
```

Comment:
What functionalities this is needed?

I think if it is about dismiss it is because we are using bootstrap 5
I think it is needed to initialize bootstrap modal first

![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/92857477/abf71572-caea-4d77-8206-5740fbe22759)
 
 Kindly check if this solution works or check how to initialize bootstrap 5 modals

Reply:
> What functionalities this is needed?
> 
> I think if it is about dismiss it is because we are using bootstrap 5 I think it is needed to initialize bootstrap modal first
> 
> ![image](https://private-user-images.githubusercontent.com/92857477/326267760-abf71572-caea-4d77-8206-5740fbe22759.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQzNTE2NzcsIm5iZiI6MTcxNDM1MTM3NywicGF0aCI6Ii85Mjg1NzQ3Ny8zMjYyNjc3NjAtYWJmNzE1NzItY2FlYS00ZDc3LTgyMDYtNTc0MGZiZTIyNzU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI5VDAwNDI1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA4ZGU2NGM0OThmNmZhMjZhODViNjNhNzFjM2M0MDVjZThlNTkwMDkwZjkxMjM2OWRkZDFiNzE2Y2U0ZTAxNDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QYfY-P9_eKu9p83iTzjgg3dZd91l0eidfHSnPP3_XsA)
> 
> Kindly check if this solution works or check how to initialize bootstrap 5 modals

- Still getting same results even though the modal is initialized
- I tried directly modifying the modal's attributes and styles, and now the cancel button works—the modal dismisses.
- Can this be considered as an alternative solution?

![2024-04-29](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162780943/0ea24abc-21f8-4730-994b-6f5e298a9436)

Reply:
You can try it like this but if it already works its okay. Kindly update the branch

![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/92857477/eb17705c-cfc4-486b-a484-0c78ee770b4f)


![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/92857477/24d76280-c671-4141-81af-7751982e4802)